### PR TITLE
NVGRE and VXLAN EncapsulatedPacketTaskOffloadEnabled parameters descr…

### PR DESCRIPTION
**Pull request by velkovb**
…iption updated.

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws306 @get-itips @tseward

Peer review needed for this PR. Thanks.

**Comment by dariomws306** [comment]
I think it needs more work. My findings:

- NvgreEncapsulatedPacketTaskOffloadEnabled and VxlanEncapsulatedPacketTaskOffloadEnabled applies to 2016.
- EncapsulatedPacketTaskOffloadEnabled applies to 2012 and 2012 R2.
- So, examples are ok, they are for 2012 and 2012 R2.

**Comment by tseward** [comment]
@dariomws306 are you indicating the examples are fine as-is or need adjustment to accommodate for 2012/2012 R2?

**Comment by dariomws306** [comment]
@tseward they are fine as-is for 2012/2012R2.

But they don't work for 2016... So I think we should leave a comment in the example description about that.

File: docset/windows/netadapter/set-netadapterencapsulatedpackettaskoffload.md [hunk 1]
```diff
@@ -271,7 +271,7 @@ Accept wildcard characters: False
 ```
 
 ### -VxlanUDPPortNumber
-{{Fill VxlanUDPPortNumber Description}}
+The port number to use for the VXLAN UDP destination port.
```
Comment (JohanFreelancer9) [inline docset/windows/netadapter/set-netadapterencapsulatedpackettaskoffload.md]:
```suggestion
Specifies the port number to use for the VXLAN UDP destination port.
```

**Comment by velkovb** [comment]
@dariomws306  This is the 2016 version article. I have not changed the examples in the 2012/2012R2 ones.

**Comment by dariomws306** [comment]
@velkovb, it's true, my mistake. So PR looks good.

**Comment by rmca14** [comment]
No Acrolinx or staging link.

**Pull request merged**